aicsimageio/writers/rgb_writer.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from pathlib import Path
+from typing import Union
+
+import dask.array as da
+import numpy as np
+from imageio import imwrite
+
+from ..exceptions import InconsistentShapeError, InvalidDimensionOrderingError
+from ..transforms import reshape_data
+from .writer import Writer
+
+###############################################################################
+
+GREYSCALE_ORDER = "YX"
+RGB_ORDER = "YXC"
+
+###############################################################################
+
+
+class RGBWriter(Writer):
```

Comment:
How about Image2dWriter, since you said that RGB is not strictly accurate?

Reply:
I'm wondering if class is the right construct here since the only method is a single static function.

Reply:
I personally like it because even though it is a `staticmethod` the interface is still enforced by the inherited writers. Plus I personally prefer seeing `RGBWriter.save` vs `save_rgb`? Idk. I could go either way.

Answering the `2DWriter`, I think I am okay with that except is seems odd to say `2D` when you can (and likely will) pass a 3D array in.

Reply:
From small discussion with others, `RGBWriter` is preferred even if it supports greyscale.